docs/src/gui/gladevcp.adoc

```diff
@@ -357,7 +357,7 @@ using a run-in-place setup.
 [[gladevcp:embedding-tab]]
 === Embedding as a Tab
 
-To do so, edit your .ini file and add to the DISPLAY and HAL sections of INI
+To do so, edit your .ini (INI) file and add to the DISPLAY and HAL sections of INI
```

Comment:
```suggestion
To do so, edit your INI file and add to the DISPLAY and HAL sections of INI
```
to have it consistent

Reply:
?

Reply:
At some point you need to introduce that .ini and INI files are the same thing.

Reply:
Then it should be introduced in every document where "INI file" and similar is used. But that should be clear to people who can operate a pc, that an INi file is a file with the extension "ini".
Even in https://linuxcnc.org/docs/devel/html/config/ini-config.html it is not mentioned, that it is a file with .ini extension...